src/core/icons/messanger.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+export const SvgMessenger = (): JSX.Element => (
+	<svg width="66" height="66" viewBox="-12 -12 90 90">
```

Comment:
The viewport for this icon is incorrect. Could you wait until this icon is added to Figma?